src/components/OrderItem/OrderItem.jsx

```diff
@@ -1,4 +1,6 @@
 import React from 'react';
+import IconButton from '@material-ui/core/IconButton';
+import DeleteIcon from '@material-ui/icons/Delete';
 import { Price } from '..';
 
 export class OrderItem extends React.Component {
```

Comment:
This doesn't need to be a class as it doesn't manage any state. It just calls `removeFromOrder` which is passed in as a prop anyway 👍

Reply:
Yeah I got linting errors for this in a few files I think man - would it be better to export at a pure function if no state is involved? don't extend `React.Component`?

Reply:
Yeah exactly man as its only bothered with props, can just destructure those in the params..

```js
export const OrderItem = ({ name, price, bread, isHot, hasButton, orderOwner }) => { // return stuff }; .......
```